canonical-websites/snapcraft.io
# Preview listing page

Fixes https://github.com/CanonicalLtd/snap-squad/issues/852
Fixes https://github.com/CanonicalLtd/snap-squad/issues/853
Fixes https://github.com/CanonicalLtd/snap-squad/issues/854

## QA
- Pull the branch
- `./run`
- Visit http://0.0.0.0:8004/<snap_name>/listing
- Click the preview button
- Switch back to the "listing" tab, make some changes, without saving
- Take a look at the "preview" tab and see the changes
- Try adding "?preview-true" to the url of a snap you don't have access to/ haven't visited the listing page for
- See an error notification
